source/_dashboards/area.markdown

```diff
@@ -84,6 +82,10 @@ features_position:
   description: Position of the features on the area card. Can be `bottom` or `inline`. Only the first feature will be displayed when the option is set to `inline`.
   type: string
   default: bottom
+exclude_entities:
+  required: false
+  description: A list of entities that will be excluded from the card. It will affect sensor_classes, alert_classes and features.
```

Comment:
```suggestion
  description: A list of entities that will be excluded from the card. It will affect sensor_classes, alert_classes, and features.
```